### PR TITLE
Update rdkafka for php7

### PR DESCRIPTION
**Pull request by MattKetmo**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

Hello, here's an update of the rdkafka extension.
https://github.com/arnaud-lb/php-rdkafka/releases

- **2.0.1** version is almost 1y old
- **3.0.4** was released in August.

Thanks.

### CODE REVIEW

File: Formula/php70-rdkafka.rb [hunk 1]
```diff
@@ -4,17 +4,10 @@ class Php70Rdkafka < AbstractPhp70Extension
...
   desc "PHP extension for Apache Kafka (php-rdkafka)"
   homepage "https://github.com/arnaud-lb/php-rdkafka"
-  url "https://pecl.php.net/get/rdkafka-2.0.1.tgz"
-  sha256 "43a3704198f7d8ae23f6ed06d2d28ae4d52bae0f93f484aa5d61d5d60f809eee"
+  url "https://pecl.php.net/get/rdkafka-3.0.4.tgz"
+  sha256 "4cb0a37664810d2a338cf5351e357be7294d99458f76435801e0ed5e328dc5ee"
   head "https://github.com/arnaud-lb/php-rdkafka.git"
 
-  bottle do
-    cellar :any
```
Comment (SMillerDev) [inline Formula/php70-rdkafka.rb]: These bottle blocks are required. They allow the build bot to make premade binaries available to everyone after your PR is accepted.

**Comment by MattKetmo** [comment]
- Commit message updated
- bottle blocks readded

Note: I didn't updated the `php5x-rdkafka` files. Should I?

File: Formula/php72-rdkafka.rb [hunk 2]
```diff
@@ -4,8 +4,8 @@ class Php72Rdkafka < AbstractPhp72Extension
   init
   desc "PHP extension for Apache Kafka (php-rdkafka)"
   homepage "https://github.com/arnaud-lb/php-rdkafka"
-  url "https://pecl.php.net/get/rdkafka-2.0.1.tgz"
-  sha256 "43a3704198f7d8ae23f6ed06d2d28ae4d52bae0f93f484aa5d61d5d60f809eee"
+  url "https://pecl.php.net/get/rdkafka-3.0.4.tgz"
+  sha256 "4cb0a37664810d2a338cf5351e357be7294d99458f76435801e0ed5e328dc5ee"
   head "https://github.com/arnaud-lb/php-rdkafka.git"
   revision 1
```
Comment (SMillerDev) [inline Formula/php72-rdkafka.rb]: This is no longer needed.

Comment (MattKetmo) [reply Formula/php72-rdkafka.rb]: `revision 1` removed ✔️

**Comment by SMillerDev** [comment]
@MattKetmo you can update 5.6 if you want to. Anything older is deprecated anyway.

**Comment by MattKetmo** [comment]
Ok so I don't touch them. They are all in an older 1.0.0 version anyway :)

**Comment by SMillerDev** [comment]
Once #4465 is merged we'll run the tests again and merge this.

**Comment by SMillerDev** [comment]
@BrewTestBot Test this please!

**Comment by SMillerDev** [comment]
@BrewTestBot Test this please!

**Comment by SMillerDev** [comment]
@BrewTestBot Test this please!

**Comment by SMillerDev** [comment]
Thanks @MattKetmo !

**Pull request closed without merging**